aws/field/field.go

```diff
@@ -36,6 +36,12 @@ const (
 	DeploymentOption // Deployment Option
 	LicenseModel     // License Model
 
+	// FSX fields
+	FileSystemType     // File system type
+	StorageType        // Storage type
+	ThroughputCapacity // Throughput capacity
+	Deployment_option  // Deployment option
```

Comment:
it should have been `DeploymentOption // Deployment option` like the other logic, but aws csv field has two different names `Deployment option` and `Deployment Option`
So instead of have `DeploymentOption` and `Deploymentoption` we choose to use `Deployment_option` to match the lowercase one